### DISCUSSION TITLE
Sqlite to etcd migration

### DISCUSSION THREAD

**Discussion by alwaysastudent** [gravitational/teleport#10402] [Q&A] [answered]
Hi, is there a way we could migrate our auth server 
storage backend from sqlite to etcd? Is there any documentation around this?

Edit: Is there a way to migrate the data from Sqlite to etcd?

**Comment by scottpgallagher** [comment]
The docs for etcd configuration can be found here: https://goteleport.com/docs/setup/reference/backends/#etcd

**Reply by alwaysastudent** [reply]
@scottpgallagher - But is there way to migrate the existing data from Sqlite to etcd as well? Sorry, I should have made my question a bit more specific.

**Comment by alwaysastudent** [comment]
Is there any option to migrate sqlite data to etcd?

**Comment by scottpgallagher** [comment, marked as answer]
Within Teleport there isn't. There might be tooling out there that does that, but a quick search didn't yield any useful results that I would recommend using.
